docs/tutorials/semantic_search/asymmetric_embedding_model.md

```diff
@@ -0,0 +1,539 @@
+# Tutorial: Running Asymmetric Semnantic Search within OpenSearch
+
+This tutorial demonstrates how to generate text embeddings using an asymmetric embedding model in OpenSearch which will be used
+to run semantic search. This is implemented within a Docker container, the example model used in this tutorial is the multilingual
```

Comment:
Can you replace "this" with a noun? Is it the search that is implemented within a Docker container?
```suggestion
to run semantic search, implemented within a Docker container. The example model used in this tutorial is the multilingual
```

Reply:
Docker isnt anything special to this tutorial usually tutorials are made with running OpenSearch locally. I figured I used docker since there aren't many tutorials using it.